CHANGELOG.md

```diff
@@ -9,10 +9,17 @@ This changelog documents all notable user-facing changes of VAST.
 ### 🧬 Experimental Features
 ### 🎁 Features
 ### 🐞 Bug Fixes
+### 🚀 Performance Improvements
```

Comment:
Can we just make this a `Feature`? Otherwise this will get really crowded in the release notes.

Reply:
I would make it a change, not a feature. Performance is typically considered a non-functional requirement.

Reply:
I think performance improvement belongs in there. I'm wondering why the options here aren't aligned with the labels in Github.

Reply:
"in there" is ambiguous. Mind clarifying?

Reply:
I mean that we should have a "performance" label in the changelog.

Reply:
I disagree. If we want to make this change, let's discuss it separately and make it part of the release prep PR.

Reply:
Alrighty.